keycloak/keycloak-admin-ui
# Users(groups): Add modal for joining user groups

## Motivation

https://issues.redhat.com/browse/APPDUX-869

## Brief Description

Add functionality to join groups 

## Verification Steps



1. Go to Users section
2. Click on groups tab
3. Click "join group" button
4. Select a group and verify the new group shows up in the data list

## Checklist:


## Additional Notes
